extensions/amp-story/1.0/amp-story.js

```diff
@@ -690,12 +690,10 @@ export class AmpStory extends AMP.BaseElement {
       return;
     }
 
-    this.paginationButtons_ = PaginationButtons.create(this.win);
+    this.paginationButtons_ =
+        PaginationButtons.create(this.win, () => this.hasBookend_());
```

Comment:
Plz add a TODO so we find a way to avoid passing a private function here. I understand that it's way out of the scope of this PR, and is already something we do in other places

Reply:
Done.